src/org/ses/android/soap/tasks/buscarHuellaTask.java

```diff
@@ -0,0 +1,70 @@
+package org.ses.android.soap.tasks;
+
+import android.os.AsyncTask;
+import android.util.Log;
+
+import org.ses.android.soap.database.Participant;
+
+import org.ksoap2.SoapEnvelope;
+import org.ksoap2.serialization.SoapObject;
+import org.ksoap2.serialization.SoapPrimitive;
+import org.ksoap2.serialization.SoapSerializationEnvelope;
+import org.ksoap2.transport.HttpTransportSE;
+
+
+
+/**
+ * Created by franciscorivera on 1/8/16.
+ */
+public class BuscarHuellaTask extends AsyncTask<Byte, Void, Participant> {
+
+    @Override
+    protected Participant doInBackground(Byte ... params)
+    {
+        Participant par = new Participant();
```

Comment:
It might help to have a more descriptive name here, like `participant` or `participante`.